Harbor/PreferencesPaneWindowController.swift

```diff
@@ -142,4 +142,24 @@ class PreferencesPaneWindowController: NSWindowController, NSWindowDelegate, NST
   private func enableOrDisableSaveButton() {
     savePreferencesButton.enabled = codeshipAPIKeyError.stringValue.isEmpty && refreshRateError.stringValue.isEmpty
   }
+
+  private func convertIntToBoolean(loginState: Int) -> Bool {
```

Comment:
I take back what I said about the extension.

I think it makes sense to add an extension `var` to `NSButton` like `var on: Bool`(or `checked` or something) that updates the button's `state` to either `NSOnState` or `NSOffState` (I'd also use those named values instead of 1/0).